source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -248,10 +248,10 @@ void AuthenticatorImpl::verifyKey() {
 
   // Forward the payload
   const auto& provider = jwks_data_->getJwtProvider();
-  std::string payload_str_base64url_with_padding = jwt_->payload_str_base64url_;
-  Base64::completePadding(payload_str_base64url_with_padding);
 
   if (!provider.forward_payload_header().empty()) {
+    std::string payload_str_base64url_with_padding = jwt_->payload_str_base64url_;
```

Comment:
variable name is too long,  can we just call it "payload_with_padding"?

Reply:
Done.